app/controllers/internal_api/v1/pillars_controller.rb

```diff
@@ -142,6 +142,7 @@ def openstack_cloud_contents
           subnet:         Pillar.value(pillar: :cloud_openstack_subnet),
           bs_version:     Pillar.value(pillar: :cloud_openstack_bs_version),
           lb_mon_retries: Pillar.value(pillar: :cloud_openstack_lb_mon_retries)
```

Comment:
Missing a comma and the end of the line.

Reply:
Fixed